hfsummary/README.md

```diff
@@ -0,0 +1,3 @@
+# indico-plugin-hf-summary
```

Comment:
Not a big fan of the name (what does hf stand for?) - how about calling the plugin "AI Summary" or something like that?